src/main/java/io/vertx/kafka/client/consumer/impl/KafkaConsumerImpl.java

```diff
@@ -389,7 +389,14 @@ public void committed(TopicPartition topicPartition, Handler<AsyncResult<OffsetA
       if (done.succeeded()) {
         // TODO: use Helper class and stream approach
         List<PartitionInfo> partitions = new ArrayList<>();
-        for (org.apache.kafka.common.PartitionInfo kafkaPartitionInfo: done.result()) {
+        List<org.apache.kafka.common.PartitionInfo> partitionInfos;
+        try {
+          partitionInfos = done.result();
+        } catch (Exception e) {
+          handler.handle(Future.failedFuture(e));
```

Comment:
Can we add tests?

Reply:
Sure.

Reply:
I have added a test for this. Can you guys have a look?